app/stac_api/urls.py

```diff
@@ -45,4 +46,5 @@
         AssetDetail.as_view(),
         name='asset-detail'
     ),
+    path(f"{API_BASE}/{HEALTHCHECK_ENDPOINT}", CollectionList.as_view(), name='collection-list')
```

Comment:
I would put this just below the landing page and change the name as `health-check` instead of 'collection-list'. In order to keep a meaningful order of the endpoint in code.